examples/panel-datalinks/package.json

```diff
@@ -18,11 +18,13 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@babel/core": "^7.21.4",
+    "@grafana/e2e-selectors": "11.2.2",
```

Comment:
same here

Reply:
Ah, missed to remove that one. Did it in the templates PR. I fix 👍🏻